fine_print.gemspec

```diff
@@ -16,13 +16,13 @@ Gem::Specification.new do |s|
   s.files = Dir['{app,config,db,lib}/**/*'] + ['MIT-LICENSE', 'Rakefile', 'README.md']
   s.test_files = Dir['spec/**/*']
 
-  s.add_dependency 'rails', '>= 4.2'
+  s.add_dependency 'rails', '>= 4.2', '< 5.2.1'
```

Comment:
Why limiting Rails to less than 5.2.1?  If it is because of the sqlite thing, I don't know if we want to make that development dependency impact the version of Rails that fine_print can support

Reply:
Yup, and my not-quite understanding Gemfile vs <app>.gemspec for a library (gem) repo . Will move the testing requirement to the Gemfile. Here's the tracking issue, seems it'll be fixed in 5.2.3 (5.2.2 and 5.2.2.1 both fail as well)
https://github.com/rails/rails/issues/35387
(Surprised that following the tutorial leads to a broken app - doesn't Rails (the project) have any automated testing, or at least smoke test the new-developer-experience?)